sdk/tables/Azure.Data.Tables/tests/TableEntityTests.cs

```diff
@@ -59,14 +59,27 @@ public void DynamicEntityGetWrongTypesThrows()
             Assert.That(() => entityWithAllTypes.GetInt64("binary"), Throws.InstanceOf<InvalidOperationException>(), "GetInt64 should validate that the value for the inputted key is a Int64.");
         }
 
+        /// <summary>
+        /// Validates getters for non-nullable types involving null. (ex. DateTime, int)
+        /// </summary>
+        [Test]
+        public void ValidateDictionaryTableEntityGetNonNullType()
+        {
+            var dummyKey = "I'm a little dumb";
```

Comment:
nit: I'd use naming to clarify intent here: `stringTypedValue`

Reply:
That variable was supposed to represent a key that doesn't exist! I'll rename it to `nonexistentKey`.